src/ragger/conftest/base_conftest.py

```diff
@@ -53,6 +53,12 @@ def pytest_addoption(parser):
                      default="default",
                      help="Specify the setup fixture (e.g., 'prod_build')",
                      choices=allowed_setups)
+    parser.addoption('--Verbose', action='store_true', default=False, help='Increase verbosity')
```

Comment:
Can we reuse the default `verbose` of pytest ?

Reply:
It would be great, but I've not found anything on this. It seems the `verbose` management is kinda hard coded into `pytest`. :disappointed: